### DISCUSSION TITLE
[Projects Beta] Drag & Drop views

### DISCUSSION THREAD

**Discussion by pavlenex** [community/community#7471] [Projects and Issues] [answered]
Sometimes I create a new view only to realize I'd like to re-arrange the order in which it appears. This isn't possible with drag and drop, it seems to me that I need to completely remove views in order to re-arrange.

**Comment by azenMatt** [comment, marked as answer]
Hi @pavlenex  - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄

**Reply by pavlenex** [reply]
Hell yeah, I saw it and already used it - keep rocking @azenMatt 🚀
